deb-x64/Dockerfile

```diff
@@ -138,6 +140,14 @@ RUN curl -sSfL -o golangci-lint-install.sh https://raw.githubusercontent.com/gol
     && echo "942efc00799b83aebaf6628c34e1151467b7fa21f0cfd78554279dc73f8c8ef3  $GOPATH/bin/golangci-lint" | sha256sum --check \
     && rm golangci-lint-install.sh
 
+# Rust is needed to compile some python libs
+RUN curl -sSL -o rustup-init.sh https://raw.githubusercontent.com/rust-lang/rustup/master/rustup-init.sh \
```

Comment:
The script [changes from time to time](https://github.com/rust-lang/rustup/commits/master/rustup-init.sh), should we pin a specific commit/tag instead of `master`?

Reply:
Yes, we should. As an example, we can reuse what's done in the [SUSE image build](https://github.com/DataDog/datadog-agent-buildimages/blob/dfc7750f76243b754297f4835dd82aa440aa8efb/suse-x64/Dockerfile#L131-L137).

Reply:
Dang, I changed to using the script from master just so we could pin it, then forgot to pin it.